src/gwt/www/unsupported_browser.htm

```diff
@@ -29,13 +29,15 @@ <h3 id="banner"><a href="http://www.rstudio.com"><img src="images/rstudio.png" w
 
 <ul>
 
-<li><a href="http://www.getfirefox.com/">Firefox 10</a></li>
-<li><a href="http://www.apple.com/safari/">Safari 5.1</a></li>
-<li><a href="http://www.google.com/chrome">Google Chrome 21</a></li>
-<li><a href="http://www.microsoft.com/en-us/download/internet-explorer.aspx">Internet Explorer 10</a></li>
+<li><a href="https://www.getfirefox.com/">Firefox 68</a></li>
+<li><a href="https://www.apple.com/safari/">Safari 12.1</a></li>
+<li><a href="https://www.google.com/chrome/">Google Chrome 71</a></li>
+<li><a href="https://www.microsoft.com/en-us/download/internet-explorer.aspx">Internet Explorer 11</a></li>
```

Comment:
At some point we'll have to add in explicit mention of (Chromium-based) Edge. They are at RC-level now, and aiming to launch in January. Your call if we should add that in now, or wait until we actually see if people start using it.

https://www.theverge.com/2019/11/4/20942038/microsoft-edge-chromium-release-date-new-logo-features

Reply:
A good question. We haven't tested with it yet so I don't think we should add it to this list. We will will want to do that later once they have a stable (supported) build.